app/services/InviteLink.js

```diff
@@ -21,6 +22,15 @@ class InviteLink extends Service {
         const fetchOptions = this.fetchOptions(data, 'POST', headers);
         return this.fetchText(url, fetchOptions);
     }
+
+    encodeURLNameParams(invitation) {
```

Comment:
Hi Megan, can you please add a unit test for _encodeURLNameParams_ so coverage is maintained, thanks.

Reply:
Sure!